### ISSUE TITLE
tor newnym issue ??

### ISSUE THREAD

**Issue by aktiftablet** [deathsec/instagram-py#41]
**Select your issue type:** (check at least one)
- [x] Bug
- [x] Question
- [ ] Suggestion
- [ ] Other (please describe):

Hello,

Works perfectly if the correct password could be guessed in the first 10 tries.
After 10 or so tries the TOR consoles startes showin notice : "Rate limiting NEWNYM request: delaying x second(s)".  During this instagram-py keeps trying the same password over and over again.

**Comment by fiebroso13** [comment]
it happens to me to it sucks

**Comment by AndresDominguezB** [comment]
Yeah it happens to me too, but i´m searching methods to speed it up. If I find anything I´ll comment it below
